src/deprecated.jl

```diff
@@ -0,0 +1,10 @@
+function checkpoint_deprecation()
```

Comment:
If we did 
```suggestion
function checkpoint_deprecation(tags...)
```
then we we can move the `isempty` here.
and can print the alternative.